Makefile

```diff
@@ -1,8 +1,16 @@
 .PHONY: setup update run
 
-setup:
+setup-execution-helper: 
```

Comment:
IMHO, I would have these separated as different tasks. This is doing 3 distinct things